src/user-guide/task-implementation/run-modes.rst

```diff
@@ -0,0 +1,211 @@
+.. _task-run-modes:
+
+Task Run Modes
+==============
+
+.. versionadded:: 8.4.0
+
+   * Before 8.4.0 run modes could only be set for the workflow, not
+     for individual tasks.
+   * Skip mode.
+
+
+As well as the default task implementation ("live" mode)
+other implementations can be selected:
+
+* By setting the scheduler run mode as an argument to the
+  ``cylc play --mode`` command ("simulation" and "dummy" modes).
+* By setting :cylc:conf:`[runtime][<namespace>]run mode` for
+  "skip" mode.
+
+#. :ref:`task-run-modes.simulation`: Intended to be used
+   when developing a workflow, the scheduler simulates
+   running a task without creating a "real" job.
+#. :ref:`task-run-modes.dummy`: Intended to be used
+   when developing a workflow, the scheduler replaces
+   the workflow scripts with dummy scripts and returns
+   to the default ("live") submission pathway.
```

Comment:
The submission "pathway" thing again - I think that kinda refers to internal implementation and might confuse users. Could be much simpler, e.g.:
```suggestion
   when developing a workflow. Instead of the real configured task scripts,
   the scheduler submits local jobs with empty (do nothing) "script" items.
```